config/initializers/zendesk_client.rb

```diff
@@ -6,4 +6,4 @@
   config.token = ENV.fetch('ZENDESK_TOKEN')
 end
 
-ZENDESK_GROUP_NAME = '3rd Line - Product Support (all teams)'
+ZENDESK_GROUP_ID = ENV.fetch('ZENDESK_NEW_TICKET_GROUP_ID')
```

Comment:
Do you need a condition check to fail gracefully if the ENV variable is not set?

Reply:
FWIW I think this should probably very non-gracefully! Don't know if this is graceful or not!

Reply:
Remind me, will this fetch throw an exception if the ENV variable is not set? If so, I am happy with it

Reply:
It throws `KeyError` exception :)